src/pages/home.tsx

```diff
@@ -117,7 +117,7 @@ const Home = () => {
         </section>
 
         <section id="sponsors" className="py-10 lg:px-16">
-          <p className="section-title">Our 2024-25 Sponsors</p>
+          <p className="section-title">Our Previous Sponsors</p>
```

Comment:
Wording nitpick/very subjective here, up to the team

```suggestion
          <p className="section-title">Our Past Sponsors</p>
```